app/helpers/edition_activity_buttons_helper.rb

```diff
@@ -52,7 +52,11 @@ def publish_button(edition)
   end
 
   def preview_button(edition)
-    link_to('Preview', preview_edition_path(edition), class: 'btn btn-primary btn-large')
+    if edition.published?
+      link_to('View this on the GOV.UK website', "#{Plek.find('www')}/#{edition.slug}", class: 'btn btn-primary btn-large')
```

Comment:
It's better to use `Plek.new.website_root`, as this uses `GOVUK_WEBSITE_ROOT` environment variable if it is set.